src/modules/settings/BreedingFilters.ts

```diff
@@ -24,25 +24,17 @@ const BreedingFilters: Record<string, FilterOption> = {
             new SettingOption('Shiny', '1'),
         ],
     ),
-    type1: new FilterOption<number>(
+    type1: new FilterOption<number[]>(
         'Type 1',
-        ko.observable(-2).extend({ numeric: 0 }),
+        ko.observable([0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17]),
```

Comment:
This should be created from PokemonType somehow, something like `GameHelper.enumNumbers(PokemonType).filter(v => v !== PokemonType.None)`

Reply:
Changed